VERSIONING.md

```diff
@@ -0,0 +1,198 @@
+# Cosign Versioning Policy
+
+This is the versioning policy regarding the stability of Cosign, both the API and CLI.
+
+**tl;dr (CLI):** semver for specific behaviors, but with deprecations.
+
+**tl;dr (API):** no guarantees at all (for now).
+
+Principles:
+
+- Provide well-versioned, very reliable things that people can write
+  scripts/code against.
+- Provide places for experimentation and rapid-development, not covered under
+  versioning guarantees.
+- Be explicit about what is and isn’t covered.
+- Better to promise a weaker guarantee and stick to it consistently than to be
+  vague about what is/isn’t covered and technically comply, or not comply at
+  all.
+
+## Cosign CLI
+
+The Cosign CLI’s version numbers are of the form `MAJOR.MINOR.PATCH`, but it is
+explicitly not covered by [semantic versioning][] guarantees. Instead, you
```

Comment:
what `[semantic versioning]` supposed to be a link to something?

Reply:
yeah forgot the link oops